Library/Subscription.cs

```diff
@@ -692,6 +693,11 @@ internal override void ReadXml(XmlTextReader reader)
                             RemainingBillingCycles = billingCycles;
                         break;
 
+                    case "remaining_pause_cycles":
+                        if (Int32.TryParse(reader.ReadElementContentAsString(), out int pauseCycles))
```

Comment:
There appears to be an issue here when travis is running the test:
```
Subscription.cs(697,85): error CS1644: Feature `declaration expression' cannot be used because it is not part of the C# 6.0 language specification
```

Reply:
I think it's because `pauseCycles` needs to be declared.

Reply:
@bhelx i can change, but this code works in c# 7.0 and higher https://blogs.msdn.microsoft.com/dotnet/2016/08/24/whats-new-in-csharp-7-0/

Reply:
I understand, but we need it to continue to work in older versions.

Reply:
Ok, I will change.